frontend/src/metabase/query_builder/components/filters/FilterPopover/FilterPopover.unit.spec.tsx

```diff
@@ -154,4 +129,46 @@ describe("FilterPopover", () => {
       });
     });
   });
+  describe("filter rendering", () => {
+    describe("no-value filters: %s filter", () => {
+      it.each(["is-null", "not-null", "is-empty", "not-empty"])(
+        "should not render picker or separator when selecting a new filter from the column dropdown",
+        async operator => {
+          setup({
+            filter: [operator, ["field", PRODUCTS.TITLE, null], null] as Filter,
+            showFieldPicker: false,
+          });
+
+          expect(
+            screen.queryByTestId("filter-popover-separator"),
+          ).not.toBeInTheDocument();
+
+          // if the "empty filter picker" boolean check fails, we will see the default filter picker
+          expect(
```

Comment:
In the future it'll be hard to say what the `"empty filter picker" boolean check` is without having a full context of this PR.
It'd be better to refer to the actual variable name related to this.